src/main/resources/camel/obs/obs-to-odoo-resource.xml

```diff
@@ -102,9 +102,9 @@
 
                 <choice>
                     <when>
-                        <spel>#{getProperty('obsQnAnsMap').containsKey(getProperty('obsQnUuid'))}</spel>
-
-                        <log loggingLevel="DEBUG" message="Obs has configured question concept" />
+                        <spel>#{getProperty('obsQnAnsMap').containsKey(getProperty('obsQnUuid')) &amp;&amp; getProperty('entity-instance').get('value') != null}</spel>
+                        <!--in case of obsgroup usage the value can be null-->
+                        <log loggingLevel="DEBUG" message="Obs has configured question concept and non null value" />
```

Comment:
We definitely have to add tests for this. @icrc-fdeniger, you can follow some existing test examples; https://github.com/mekomsolutions/eip-routes-odoo/blob/master/src/test/java/com/mekomsolutions/eip/route/obs/ObsToOdooResourceRouteTest.java